<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquitWebServer::AccessControlWidget</class>
 <widget class="QWidget" name="EquitWebServer::AccessControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="ipPolicyList">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="listButtonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-add" resource="../resources/resources.qrc">
         <normaloff>:/icons/buttons/addtolist</normaloff>:/icons/buttons/addtolist</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="separator">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="defaultPolicyLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="defaultPolicyLabel">
       <property name="text">
        <string>Default policy</string>
       </property>
       <property name="buddy">
        <cstring>defaultPolicy</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="EquitWebServer::ConnectionPolicyCombo" name="defaultPolicy">
       <property name="toolTip">
        <string>&lt;p&gt;Choose a default policy to use for HTTP connections from IP addresses not listed above.&lt;/p&gt;

&lt;p&gt;&lt;small&gt;This includes IP addresses for which the policy &lt;strong&gt;No Policy&lt;/strong&gt; has been set.&lt;/small&gt;&lt;/p&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EquitWebServer::ConnectionPolicyCombo</class>
   <extends>QComboBox</extends>
   <header>src/connectionpolicycombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ipPolicyList</tabstop>
  <tabstop>add</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>defaultPolicy</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
